mesa/testProton.py

```diff
@@ -74,20 +74,16 @@ def test_generate_households():
             assert len([x for x in agent.neighbors["parent"] if x.gender_is_male == False]) <= 1
 
 def test_weddings():
-    m = MesaPROTON_OC()
-    m.initial_agents = 500
-    m.create_agents(random_relationships=True, exclude_partner_net=True)
-    print(len(m.schedule.agents) - len(pp.Person.persons))
-    print(m.number_weddings)
-    m.number_weddings_mean = 100
-    for i in range(1, 2):
-        m.wedding()
-    # print(Person.NumberOfLinks()-l)
-    for agent in m.schedule.agents:
+    model = MesaPROTON_OC()
+    model.setup(1000)
+    initial_wedding = len([x for x in model.schedule.agents if x.get_neighbor_list("partner")])
```

Comment:
Once we touch it, let's move this altogether in the test routine? If I understand what it does?

Reply:
initial_wedding (line 79) basically corrects for the couples created during setup. what do you mean by "move this altogether in the test routine?"